docs/rules/class-methods-use-this.md

```diff
@@ -0,0 +1,85 @@
+# Enforce that class methods utilize `this` (class-methods-use-this)
+
+If a class method does not use `this`, it can safely be made a static function.
+
+Its possible to have a class method which doesn't use `this`, such as:
+
+```js
+class A {
+    constructor() {
+        this.a = "hi";
+    }
+
+    print() {
+        console.log(this.a);
+    }
+
+    sayHi() {
+        console.log("hi");
+    }
+}
+
+let aObj = new A();
+a.sayHi(); // => "hi"
+```
+
+Above example shows that `sayHi` method doesn't uses `this`, so we can easily make that as a static function, like this:
+
+```js
+class A {
+    constructor() {
+        this.a = "hi";
+    }
+
+    print() {
+        console.log(this.a);
+    }
+
+    static sayHi() {
+        console.log("hi");
+    }
+}
+
+A.sayHi(); // => "hi"
+```
+
+## Rule Details
+
+This rule is aimed to flag class methods that do not use `this`.
+
+Examples of **incorrect** code for this rule:
+
+```js
+/*eslint class-methods-use-this: 2*/
```

Comment:
From a quick glance over the docs, it looks like we're using the string levels ("off"/"warn"/"error") in code examples now.